content/es/docs/contribute/localization_es.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Contribuir a la documentación de Kubernetes en español
+content_type: concept
+---
+
+<!-- overview -->
+
+¡Bienvenida/o! Esta página es un esbozo.
+
+
+<!-- body -->
+
+## Glosario de terminología {#terminologia}
+
+|      English      |        Español         |  Género   |    Commentarios          |
+| ----------------- | ---------------------- | --------- | ------------------------ |
+| availability zone | zona de disponibilidad |           |                          |
+| bearer token      | ⚠ ?                    |           |                          |
+| built-in          | incorporados           |           |                          |
+| conditions        | estado                 |           | para node conditions     |
+| container         | contendador            |           |                          |
+| controller        | controlador            |           |                          |
+| deploy            | desplegar              |           |                          |
+| Deployment        | Deployment             |           | objeto Kubernetes        |
+| Endpoints         | Endpoints              |           | objeto Kubernetes        |
+| file              | fichero                |           |                          |
+| frontend          | frontend               |           |                          |
+| healthy           | operativo              |           |                          |
+| high availability | alta disponibilidad    |           |                          |
+| hook              | hook                   |           |                          |
+| instance          | instancia              |           |                          |
+| Lease             | Lease                  |           | objeto Kubernetes        |
+| Pod               | Pod                    | masculino | objeto Kubernetes        |
+| ratio             | ritmo                  |           |                          |
```

Comment:
```suggestion
| ratio             | radio                  |           |                          |
```

Reply:
Sobre esta no estoy muy seguro, `ratio` se refiere a una `proporción` en este caso.

Reply:
y su genero seria `femenino`

Reply:
I'm happy to include one of these suggestions - which should it be?

Reply:
```suggestion
| ratio             | proporción             | femenino  |                          |
```